install.sh

```diff
@@ -12,12 +12,32 @@ for ip in $internal_ips; do
 done
 
 # Check for root privileges
-if [ "$(whoami)" != "root" ]
-  then
-  log ""
-  log "Error installing reNgine-ng: please run this script as root!" $COLOR_RED
+if [ $EUID -eq 0 ]; then
+  if [ "$SUDO_USER" = "root" ] || [ "$SUDO_USER" = "" ]; then
+    log "Error: Do not run this script as root. Use sudo with a non-root user." $COLOR_RED
```

Comment:
```suggestion
    log "Error: Do not run this script as root user. Use `sudo` with a non-root user." $COLOR_RED
```

Backticks should work as it's within `log`. And is it worth mentioning that the non-root user should be in `sudoers`?

Reply:
Not sure but why do not put simple quote I stead of back tick ?

Reply:
Replaced by single quote